src/main/java/eu/bbmri_eric/negotiator/database/model/Network.java

```diff
@@ -1,134 +1,99 @@
 package eu.bbmri_eric.negotiator.database.model;
 
-import jakarta.persistence.CascadeType;
-import jakarta.persistence.Column;
-import jakarta.persistence.Entity;
-import jakarta.persistence.FetchType;
-import jakarta.persistence.GeneratedValue;
-import jakarta.persistence.GenerationType;
-import jakarta.persistence.Id;
-import jakarta.persistence.JoinColumn;
-import jakarta.persistence.JoinTable;
-import jakarta.persistence.ManyToMany;
-import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.*;
 import jakarta.validation.constraints.NotNull;
-import java.util.Collections;
 import java.util.HashSet;
 import java.util.Objects;
 import java.util.Set;
-import lombok.AccessLevel;
-import lombok.AllArgsConstructor;
-import lombok.Builder;
-import lombok.Getter;
-import lombok.NoArgsConstructor;
-import lombok.Setter;
-import lombok.ToString.Exclude;
+import lombok.*;
 
-/**
- * Represents a Network entity in the database. A Network has a unique identifier, a URI, a name, a
- * contact email, a set of managers, and a set of resources.
- */
 @Getter
 @Setter
 @Entity
 @Builder
-@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
```

Comment:
Why not protected?

Reply:
please rebase this to latest master

Reply:
I think you have mix of old and new commits, please fix it

Reply:
_"The JPA specification requires all Entity classes to have a default no-arg constructor. This can be either public or protected."_

https://www.baeldung.com/jpa-no-argument-constructor-entity-class